contrib/alacritty-terminfo

```diff
@@ -0,0 +1 @@
+alacritty
```

Comment:
i forget if it's required, but shouldn't you depends=alacritty-terminfo=pkgver-rpkgrel on this from the base package? it's normally separate to allow standalone installation of the terminfo on another machine, but the terminal itself would need it